ext/redhat/razor-server.env

```diff
@@ -7,4 +7,5 @@ JBOSS_CONFIG=standalone.xml
 JBOSS_MODULES_SYSTEM_PKGS=org.jboss.byteman
 LAUNCH_JBOSS_IN_BACKGROUND=true
 LANG=en_US.UTF-8
-JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
\ No newline at end of file
+JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
```

Comment:
The "preferIPv4Stack" option sounds like it'll still let you use IPv6, but in fact it [disables IPv6 entirely](http://download.java.net/jdk7/archive/b123/docs/api/java/net/doc-files/net-properties.html). Is this what you want? Will folks never want to use razor on IPv6-only hosts?